src/data/sql-quiz.ts

```diff
@@ -439,6 +439,29 @@ const sqlQuiz = [
       "The ROLLBACK TRANSACTION command will erase all modifications made to the data. Changes made to table and local variables will not be affected by this command.",
     Link: "https://learn.microsoft.com/en-us/sql/t-sql/language-elements/rollback-transaction-transact-sql?view=sql-server-ver16",
   },
+  {
+    Question:
+      "Can we join two Tables without foriegn key?",
+    Answer: "NO",
+    Distractor1: "YES",
+    Distractor2: "MAYBE",
+    Distractor3: "NOT SURE",
+    Explanation:
+      "The REVOKE statement can be used to undo the action of a GRANT statement.",
+    Link: "https://dev.mysql.com/doc/refman/5.6/en/revoke.html",
```

Comment:
We try to avoid questions that only have true or false answers.
We want to add questions that have more options for the distractor fields than yes, no or maybe.
Also the explanation looks like it is from another quiz questions.

Let's remove this question. 👍